examples/hello_world/client/nodejs/index.js

```diff
@@ -1,24 +1,54 @@
 const fs = require('fs');
-const protoLoader = require('@grpc/proto-loader');
+const protobufjs = require('protobufjs');
 const grpc = require('@grpc/grpc-js');
+const grpcProtoLoader = require('@grpc/proto-loader');
 
 const CERT_PATH = __dirname + '/../../../certs/local/ca.pem';
-const PROTO_PAH = __dirname + '/../../proto/hello_world.proto';
-
-const packageDefinition = protoLoader.loadSync(PROTO_PAH);
-const helloWorldProto = grpc.loadPackageDefinition(packageDefinition).oak
-  .examples.hello_world;
-const credentials = grpc.credentials.createSsl(fs.readFileSync(CERT_PATH));
-
-// TODO(#1066): Use a more restrictive Label.
-const client = new helloWorldProto.HelloWorld('localhost:8080', credentials);
-
-client.sayHello({ greeting: 'Node.js' }, (error, response) => {
-  if (error) {
-    console.error(error);
-    process.exit(1);
-  } else {
-    console.log(response.reply);
-    process.exit(0);
+const SERVICE_PROTO_PAH = __dirname + '/../../proto/hello_world.proto';
+const OAK_LABEL_PROTO_PAH = __dirname + '/../../../../oak/proto/label.proto';
+
+// Keep in sync with /oak/server/rust/oak_runtime/src/node/grpc/server/mod.rs.
+const oakLabelGrpcMetadataKey = 'x-oak-label-bin';
+
+async function main() {
+  const [oaklabelDefinition, helloWorldDefinition] = await Promise.all([
+    protobufjs.load(OAK_LABEL_PROTO_PAH),
+    grpcProtoLoader.load(SERVICE_PROTO_PAH),
```

Comment:
We end up using two different libs with a similar API to load proto files, which feels a bit odd. 

`@grpc/proto-loader` is a wrapper around `protobufjs`, but sadly they can't be used interchangeably. It's a part of the design of the node implementation of gRPC though, so we'll have to live with it.

Reply:
It may be good to capture this in a comment in the actual source?

Reply:
Can do. :)